src/common/storage.js

```diff
@@ -0,0 +1,118 @@
+import { request, buffer2string, ensureArray } from '#/common';
```

Comment:
or maybe just `'.'`?

Reply:
`'.'` is ok if it does not cause circular dependency. Maybe better to move the methods to a file other than `index.js`. 😅